package.json

```diff
@@ -5,10 +5,12 @@
   "main": "index.js",
   "scripts": {
     "lint": "npm run lint:js && npm run lint:sass",
-    "lint:js": "eslint --ext .js,.vue,.html calendar datepicker tests",
+    "lint:js": "eslint --ext .js,.vue,.html calendar datepicker tests docs",
     "lint:sass": "sass-lint -q -v",
     "pretest": "npm run lint",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "npm run lint && npm run docs:generate",
```

Comment:
Because `pretest` is set to run `npm run lint`, that doesn't need to be repeated as part of `test`. NPM will run `pretest` and ensure it passes before running `test`

Reply:
Sure